packages/aila/src/core/chat/AilaStreamHandler.ts

```diff
@@ -81,9 +83,15 @@ export class AilaStreamHandler {
     log.info("Starting stream", { chatId: this._chat.id });
     this.setupController(controller);
     try {
-      log.info("Setting up generation");
-      await this._chat.setupGeneration();
-      this.logStreamingStep("Setup generation complete");
+      if (!this._chat.aila.options.useAgenticAila) {
+        log.info("Setting up generation");
+        await this._chat.setupGeneration();
+        this.logStreamingStep("Setup generation complete");
+      } else {
+        log.info("Initialising Chunks");
+        this._chat.initialiseChunks();
```

Comment:
initialiseChunks() was called in setupGeneration() ... but we no lnoger use the generation service in Agentic Aila